ui/apps/platform/package.json

```diff
@@ -20,6 +20,7 @@
         "@patternfly/react-table": "4.112.39",
         "@patternfly/react-topology": "4.93.5",
         "@patternfly/react-user-feedback": "^1.1.2",
+        "@types/cypress": "^1.1.3",
```

Comment:
Do we need to add this dependency? When I was messing around with trying to get this to work with the component tests, it looks like the types were shipped directly with the main Cypress dependency.